monitoring/Dockerfile

```diff
@@ -72,7 +72,31 @@ COPY --from=c1 /usr/share/grafana /usr/share/grafana
 COPY --from=c1 $GF_PATHS_HOME/bin/grafana-cli $GF_PATHS_HOME/bin/grafana-cli
 COPY --from=c1 $GF_PATHS_HOME/public $GF_PATHS_HOME/public
 
-RUN grafana-cli plugins install grafana-image-renderer
+# Grafana image rendering borrowed from https://github.com/grafana/grafana/blob/main/packaging/docker/custom/Dockerfile
+RUN apk add --no-cache libaio libnsl && \
+    ln -s /usr/lib/libnsl.so.2 /usr/lib/libnsl.so.1 && \
+    wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.30-r0/glibc-2.30-r0.apk \
+      -O /tmp/glibc-2.30-r0.apk && \
+    wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.30-r0/glibc-bin-2.30-r0.apk \
+      -O /tmp/glibc-bin-2.30-r0.apk && \
+    apk add --no-cache --allow-untrusted /tmp/glibc-2.30-r0.apk /tmp/glibc-bin-2.30-r0.apk && \
+    rm -f /tmp/glibc-2.30-r0.apk && \
+    rm -f /tmp/glibc-bin-2.30-r0.apk && \
+    rm -f /lib/ld-linux-x86-64.so.2 && \
+    rm -f /etc/ld.so.cache;
```

Comment:
This doesn't seem to come from the `Dockerfile` you mentioned. 

Maybe include the src for this one here as well and/or move the "borrowed from" comment to the command below?

Reply:
Done 631c51c